src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -120,8 +132,11 @@ public Person toModelType() throws IllegalValueException {
         }
         final GradeList modelGradeList = new GradeList(convertedGrades);
 
-        // TODO: Store attendanceList properly
-        final AttendanceList modelAttendancelist = new AttendanceList();
+        final Map<LocalDateTime, Attendance> convertedAttendances = new TreeMap<>();
+        for (Map.Entry<LocalDateTime, JsonAdaptedAttendance> entry : attendances.entrySet()) {
+            convertedAttendances.put(entry.getKey(), entry.getValue().toModelType());
```

Comment:
Would following the same implementation as [line 81](https://github.com/AY2425S1-CS2103T-T08-4/tp/pull/65/files#diff-c4705f0e6c86ae58f17f0dd1cec5b1abc7614ae894187cfe901068b6d6fb012dR81) to convert an entry set to map be more consistent?